client/messaging/message-layout.tsx

```diff
@@ -44,6 +44,8 @@ const longTimestamp = new Intl.DateTimeFormat(navigator.language, {
 export const Separator = styled.i`
   position: absolute;
   display: inline-block;
+  top: -10000px;
```

Comment:
I think doing this screws up the copy-pasteability, I'm fairly certain I tried this before and it did. So, test that before adding this randomly please :)

Reply:
I did test it and didn't notice any issues from what I can tell.